src/seedu/addressbook/util/FilePathUtil.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.util;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This is a utility class for file path validation.
+ */
+public class FilePathUtil {
```

Comment:
As we limit level 1 to just one class, this class should be merged back to AddressBook class.